status-server/src/public/example.html

```diff
@@ -9,11 +9,11 @@
 
   <section class="container" id="visualization">
     <div id="band">
-      <canvas id="led0" class="led" width=40 height=40></canvas>
-      <canvas id="led1" class="led" width=40 height=40></canvas>
-      <canvas id="led2" class="led" width=40 height=40></canvas>
-      <canvas id="led3" class="led" width=40 height=40></canvas>
-      <canvas id="led4" class="led" width=40 height=40></canvas>
+      <canvas id="led0" class="led" width=40 height=40 ></canvas>
+      <canvas id="led1" class="led" width=40 height=40 onclick="parent.animate(0);"></canvas>
```

Comment:
You shouldn't need to be calling functions in the parent window, this functionality can go in client.js which is included by each example.html. In fact as far as I can tell, ClientPage.html doesnt need any script at all. 
Perhaps this function would be better named "sendClick"?